android/app/src/main/AndroidManifest.xml

```diff
@@ -53,6 +53,22 @@
                 <action android:name="android.intent.action.MAIN"/>
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
+
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" android:host="lichess.org" />
+
+                <data android:pathPattern="/training/....." />
```

Comment:
Might as well add `streak` and `storm` since they are fully implemented on the app?

Reply:
why not, done :+1: